docker-centos/Dockerfile

```diff
@@ -7,8 +7,8 @@ LABEL maintainer="Giuseppe Scrivano <gscrivan@redhat.com>" \
       architecture="x86_64"
 
 
-RUN yum install --setopt=tsflags=nodocs -y docker container-selinux cloud-utils-growpart python-docker-py docker-novolume-plugin docker-lvm-plugin lvm2 iptables procps-ng xz oci-register-machine && \
-rpm -V docker container-selinux cloud-utils-growpart python-docker-py docker-novolume-plugin docker-lvm-plugin lvm2 iptables procps-ng xz oci-register-machine && \
+RUN yum install --setopt=tsflags=nodocs -y docker container-storage-setup container-selinux cloud-utils-growpart python-docker-py docker-novolume-plugin docker-lvm-plugin lvm2 iptables procps-ng xz oci-register-machine && \
+rpm -V docker container-storage-setup container-selinux cloud-utils-growpart python-docker-py docker-novolume-plugin docker-lvm-plugin lvm2 iptables procps-ng xz oci-register-machine && \
 mkdir -p /usr/lib/modules && \
 yum-config-manager --nogpgcheck --add-repo https://cbs.centos.org/repos/virt7-docker-el-candidate/x86_64/os/ && \
```

Comment:
@giuseppe you should change the koji tag to virt7-docker-common-candidate. I've stopped building packages for -el-candidate, unless of course, someone else is continuing to build here.

Since docker repos were a little different for rhel and fedora, i did request for -el-candidate koji tags in CentOS, but now they're mostly unified, so -common-candidate is where I send builds.